CHANGES.rst

```diff
@@ -23,6 +23,10 @@ New Features
   - Added ``PrecessedGeocentric`` frame, which is based on GCRS, but precessed
     to a specific requested mean equinox. [#3758]
 
+  - `SphericalRepresentation` now has a `._unitrep` class attribute to specify 
+    an equivalent UnitSphericalRepresentation. This allows subclasses of 
+    representations to pair up correctly.
```

Comment:
Should this really go here if it's describing an attribute for internal-only use, or is this something authors of `SphericalRepresentation` subclasses need to be aware of?